dashboards/MySQL_InnoDB_Metrics.json

```diff
@@ -1455,8 +1569,7 @@
             "titleSize": "h6"
         }
     ],
-    "schemaVersion": 13,
-    "sharedCrosshair": true,
```

Comment:
@askomorokhov seems like refactored in https://github.com/grafana/grafana/issues/6894